packages/elements-dev-portal/src/hooks/useGetBranches.ts

```diff
@@ -8,7 +8,7 @@ import { getBranches } from '../handlers/getBranches';
 export function useGetBranches({ projectId }: { projectId: string }) {
   const { platformUrl, platformAuthToken } = React.useContext(PlatformContext);
   return useQuery(
-    [...devPortalCacheKeys.branchesList(projectId), platformUrl, platformAuthToken],
```

Comment:
is there a user id or some other context we can use?

Reply:
or like if the user is actually logged out or not?

Reply:
@daniel-white We do have `state.auth.isLoggedIn` that we can pass through to the provider.  Would that work?

Reply:
Ok added the `isLoggedIn` context here https://github.com/stoplightio/elements/pull/2550/commits/9db422a2dcac03c3392f2f775c899d5ada11cecf

Reply:
Is that going to be an issue for anonymous users who never log in?

Reply:
@brendarearden I just tested it and it works as an anonymous user.  My understanding is the array being passed in to `useQuery` is used as a key and invalidated if any of those values change